components/layout/DocsLayout.tsx

```diff
@@ -146,7 +145,7 @@ export default function DocsLayout({ post, navItems = {}, children }: IDocsLayou
                 </div>
               )}
 
-              <AnnouncementHero className='ml-6' hideVideo={true} />
+              {/* <AnnouncementHero className='ml-6' hideVideo={true} /> */}
```

Comment:
I think we should add a comment something like `// TODO` so that we can find all such usages